docs/models/basics.mdx

```diff
@@ -106,6 +106,7 @@ See [dataset](../dataset/basics) to learn more about sample inputs.
 
 - `AZURE_OPENAI_API_KEY`: This is the API key to authenticate with Azure. See [their docs](https://learn.microsoft.com/en-us/javascript/api/overview/azure/openai-readme?view=azure-node-preview#using-an-api-key-from-azure) to get the API key.
 - `AZURE_OPENAI_RESOURCE_NAME`: This the resource name which is used to create the endpoint with the format `https://$AZURE_OPENAI_RESOURCE_NAME.openai.azure.com`
+- `AZURE_OPENAI_BASE_URL`: This is the base URL of the Azure OpenAI endpoint. If you have a different base URL, you can use this to set this environment variable instead of `AZURE_OPENAI_RESOURCE_NAME`.
```

Comment:
thanks for the contribution. i think it can confusing for a user that only one of these need to be specified. my suggestion would be
1. move these 2 options to a new heading: `#### Specific base URL`
2. one line intro that only one of the following need to be specified
3. one example for the base url

Reply:
Makes sense. Added changes to address these. Couldn't think of a better example than `https://some-custom-url.com` lol.

Reply:
thanks! i made a typo in my suggestion above. i've committed that fix.